2025-04-13 10:19:09,400 INFO    MainThread:100 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_setup.py:_flush():76] Configure stats pid to 100
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_setup.py:_flush():76] Loading settings from /home/git/AI-trainings/visual-language-model/wandb/settings
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_init.py:_log_setup():521] Logging user logs to /home/git/AI-trainings/visual-language-model/wandb/run-20250413_101909-qszolp09/logs/debug.log
2025-04-13 10:19:09,401 INFO    MainThread:100 [wandb_init.py:_log_setup():522] Logging internal logs to /home/git/AI-trainings/visual-language-model/wandb/run-20250413_101909-qszolp09/logs/debug-internal.log
2025-04-13 10:19:09,402 INFO    MainThread:100 [wandb_init.py:init():561] calling init triggers
2025-04-13 10:19:09,402 INFO    MainThread:100 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'epochs': 3, 'batch_size': 20, 'lr': 1.1999999999999999e-05}
2025-04-13 10:19:09,402 INFO    MainThread:100 [wandb_init.py:init():586] re-initializing run, found existing run on stack: ulqbd04a
2025-04-13 10:19:09,404 INFO    MainThread:100 [wandb_run.py:_finish():2064] finishing run amrkmrc-capgemini/vision-language-model-image-caption/ulqbd04a
2025-04-13 10:19:09,405 INFO    MainThread:100 [jupyter.py:save_history():445] not saving jupyter history
2025-04-13 10:19:09,405 INFO    MainThread:100 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2025-04-13 10:19:09,405 INFO    MainThread:100 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2025-04-13 10:19:09,405 INFO    MainThread:100 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2025-04-13 10:19:09,405 INFO    MainThread:100 [wandb_run.py:_restore():2287] restore
2025-04-13 10:19:09,406 INFO    MainThread:100 [wandb_run.py:_restore():2293] restore done
2025-04-13 10:19:15,936 INFO    MainThread:100 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2025-04-13 10:19:15,936 INFO    MainThread:100 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2025-04-13 10:19:15,951 INFO    MainThread:100 [wandb_run.py:_footer_sync_info():3895] logging synced files
2025-04-13 10:19:16,162 INFO    MainThread:100 [wandb_init.py:init():611] starting backend
2025-04-13 10:19:16,162 INFO    MainThread:100 [wandb_init.py:init():615] setting up manager
2025-04-13 10:19:16,165 INFO    MainThread:100 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-13 10:19:16,166 INFO    MainThread:100 [wandb_init.py:init():623] backend started and connected
2025-04-13 10:19:16,183 INFO    MainThread:100 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2025-04-13 10:19:16,183 INFO    MainThread:100 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-04-13 10:19:16,184 INFO    MainThread:100 [wandb_init.py:init():715] updated telemetry
2025-04-13 10:19:16,198 INFO    MainThread:100 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2025-04-13 10:19:17,949 INFO    MainThread:100 [wandb_run.py:_on_init():2357] communicating current version
2025-04-13 10:19:18,055 INFO    MainThread:100 [wandb_run.py:_on_init():2366] got version response upgrade_message: "wandb version 0.19.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-04-13 10:19:18,056 INFO    MainThread:100 [wandb_init.py:init():799] starting run threads in backend
2025-04-13 10:19:22,435 INFO    MainThread:100 [wandb_run.py:_console_start():2335] atexit reg
2025-04-13 10:19:22,435 INFO    MainThread:100 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2025-04-13 10:19:22,435 INFO    MainThread:100 [wandb_run.py:_redirect():2255] Wrapping output streams.
2025-04-13 10:19:22,435 INFO    MainThread:100 [wandb_run.py:_redirect():2280] Redirects installed.
2025-04-13 10:19:22,436 INFO    MainThread:100 [wandb_init.py:init():842] run started, returning control to user process
